docs/headless-cms/docs/03-01-docstopic-cr-lifecycle.md

```diff
@@ -0,0 +1,40 @@
+---
+title: DocsTopic custom resource lifecycle
+type: Details
+---
+
+## Asset CR manual changes
+
+The DocsTopic custom resource (CR) coordinates Asset CR creation, deletion, and changes. The DocsTopic Controller verifies DocsTopic definition on a regular basis and adds, removes, or modifies the Assets CRs accordingly.
+
+The DocsTopic CR acts as the only source of truth for the Asset CRs it orchestrates. If you modify or remove any of them manually, DocsTopic Controller automatically overwrites it or recreates it based on the DocsTopic CR definition.
```

Comment:
please specify what `recreates` means. My understanding after reading this sentence is that the CR is recreated but I tried (modified DocsTopic in one type to have different filter and url, so the content in fact is totally different) and it is not, it is just modified, the CR stays, although its old content is wiped out -> this is very important info

Reply:
Yes, that true. CR is updated, not recreated